packages/generation/ir/ir-types/src/auth.yml

```diff
@@ -2,9 +2,10 @@
 
 imports:
   types: types.yml
-  commons: ./commons.yml
+  commons: commons.yml
+  http: services/http.yml
 types:
-  AuthSchemeName: string
+  AuthSchemeName: commons.StringWithAllCasings
```

Comment:
why does the auth scheme name need to be `camel cased/ snake cased` ?

Reply:
yeah good point, we can just use string here and enforce a good name in the linter

Reply:
cool and the name there would be UpperCamelCase just like types or error names